src/App.js

```diff
@@ -53,24 +57,32 @@ class Landing extends React.Component {
   }
 }
 
-function SignIn() {
+class SignIn extends React.Component {
```

Comment:
Like @alejandronanez said, you should get used to using functional components. It's a bit strange to me to see a component go from function _to_ class based here. If you are looking to write event handlers in the component, you can do so like this:
```
function SignIn() {
  function handleClickSetLocalToken() {
  }

  return < ... />
}
```